hieradata/common.yaml

```diff
@@ -574,7 +574,7 @@ govuk_ci::master::pipeline_jobs:
   govuk-diff-pages: {}
   govuk_admin_template: {}
   govuk_ab_testing: {}
-  govuk_client_url-arbiter: {}
+  govuk-client-url_arbiter: {}
```

Comment:
It's still being used by info-frontend. 🙁 I'm not sure what the arbiter was, but I think the gem is still being used to parse API responses.

I suspect it wouldn't be very hard to remove from info-frontend, but it's not completely trivial (at least, not for someone who hasn't used url-arbiter or info-frontend before) so I'm trying to update it because it seemed like an easier yak to shave. That might not be true, though, so I'll take another look at removing it tomorrow.